.vsts-ci/linkcheck.yml

```diff
@@ -0,0 +1,44 @@
+name: PR-$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:.rr)
+trigger:
+  branches:
+    include:
+    - master
+    - release*
+  paths:
+    exclude:
+    - /src/*
```

Comment:
Why not use "include *.md" instead of "exclude" ?

Reply:
Good point, I will.

Reply:
Please don't.. wildcards only work if they are the last character

Reply:
Can you add this to the spelling?   I can do a rename later.

Reply:
Doing now.